<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Build</class>
 <widget class="QWidget" name="Build">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1276</width>
    <height>788</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="addQ">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>680</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Add New Survey Question</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1241</width>
     <height>661</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QFrame" name="frame3">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>2</number>
      </property>
      <widget class="QStackedWidget" name="stack3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>611</width>
         <height>121</height>
        </rect>
       </property>
       <widget class="QWidget" name="page_7"/>
       <widget class="QWidget" name="page_8"/>
      </widget>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QFrame" name="frame8">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>2</number>
      </property>
      <widget class="QStackedWidget" name="stack8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>611</width>
         <height>121</height>
        </rect>
       </property>
       <widget class="QWidget" name="page_17"/>
       <widget class="QWidget" name="page_18"/>
      </widget>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QFrame" name="frame7">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>2</number>
      </property>
      <widget class="QStackedWidget" name="stack7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>611</width>
         <height>121</height>
        </rect>
       </property>
       <widget class="QWidget" name="page_15"/>
       <widget class="QWidget" name="page_16"/>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="frame2">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>2</number>
      </property>
      <widget class="QStackedWidget" name="stack2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>611</width>
         <height>121</height>
        </rect>
       </property>
       <widget class="QWidget" name="page_5"/>
       <widget class="QWidget" name="page_6"/>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QFrame" name="frame1">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>2</number>
      </property>
      <widget class="QStackedWidget" name="stack1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>611</width>
         <height>121</height>
        </rect>
       </property>
       <widget class="QWidget" name="page_3"/>
       <widget class="QWidget" name="page_4"/>
      </widget>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QFrame" name="frame9">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>2</number>
      </property>
      <widget class="QStackedWidget" name="stack9">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>611</width>
         <height>121</height>
        </rect>
       </property>
       <widget class="QWidget" name="page_19"/>
       <widget class="QWidget" name="page_20"/>
      </widget>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QFrame" name="frame6">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>2</number>
      </property>
      <widget class="QStackedWidget" name="stack6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>611</width>
         <height>121</height>
        </rect>
       </property>
       <widget class="QWidget" name="page_13"/>
       <widget class="QWidget" name="page_14"/>
      </widget>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QFrame" name="frame5">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>2</number>
      </property>
      <widget class="QStackedWidget" name="stack5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>611</width>
         <height>121</height>
        </rect>
       </property>
       <widget class="QWidget" name="page_11"/>
       <widget class="QWidget" name="page_12"/>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QFrame" name="frame0">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>2</number>
      </property>
      <widget class="QStackedWidget" name="stack0">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>611</width>
         <height>121</height>
        </rect>
       </property>
       <widget class="QWidget" name="page"/>
       <widget class="QWidget" name="page_2"/>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QFrame" name="frame4">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>2</number>
      </property>
      <widget class="QStackedWidget" name="stack4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>611</width>
         <height>121</height>
        </rect>
       </property>
       <widget class="QWidget" name="page_9"/>
       <widget class="QWidget" name="page_10"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
